examples/react-alt/js/app.jsx

```diff
@@ -0,0 +1,176 @@
+/*jshint quotmark:false */
+/*jshint white:false */
+/*jshint trailing:false */
+/*jshint newcap:false */
+/*global React, Router*/
+var app = app || {};
+
+(function () {
+	'use strict';
+
+	app.ALL_TODOS = 'all';
+	app.ACTIVE_TODOS = 'active';
+	app.COMPLETED_TODOS = 'completed';
+	var TodoFooter = app.TodoFooter;
+	var TodoItem = app.TodoItem;
+	var TodoActions = app.todoActions;
+	var TodoStore = app.todoStore;
+
+	var ENTER_KEY = 13;
+
+	var TodoApp = React.createClass({
+		getInitialState: function () {
+			return TodoStore.getState();
+		},
+
+		componentDidMount: function () {
+			TodoStore.listen(this.onStoreChange);
+
+			var router = Router({
+				'/': function () {TodoActions.show(app.ALL_TODOS);},
```

Comment:
Could you add a bit of whitespace here? `function () { TodoActions.show(app.ALL_TODOS); }`. Alternatively, binding the function is fine, too: `TodoActions.show.bind(TodoActions, app.ALL_TODOS);`